tests/integration/environment/test_environment.py

```diff
@@ -258,3 +261,100 @@ def test_cmc_atmosphere(mock_show, example_spaceport_env):  # pylint: disable=un
     """
     example_spaceport_env.set_atmospheric_model(type="Ensemble", file="CMC")
     assert example_spaceport_env.all_info() is None
+
+
+@pytest.fixture
```

Comment:
since this is a fixture, it is not positioned in the right file. We should look for a better place for this.